LICENSE-3rdparty.csv

```diff
@@ -11,6 +11,7 @@ miniutf,https://github.com/dropbox/miniutf,MIT,"Copyright (c) 2013 Dropbox, Inc.
 Serilog,https://github.com/serilog/serilog,Apache-2.0,"Copyright (c) 2013-2018 Serilog Contributors"
 Serilog.Sinks.File,https://github.com/serilog/serilog-sinks-file,Apache-2.0,"Copyright (c) 2016 Serilog Contributors"
 opentracing-contrib/csharp-netcore,https://github.com/opentracing-contrib/csharp-netcore,Apache-2.0,
+opentelemetry-dotnet-instrumentation,https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation,Apache-2.0,
```

Comment:
Do we use or distribute OpenTelemetry code?

Reply:
Did we use it in the _Datadog.Tracing.OpenTelemetry_ Library?

Reply:
You mean `Datadog.Tracing.OpenTracing`. Different standard 😉

Reply:
We have started back-porting code contributed to the opentelemetry-dotnet-instrumentation repo, and I anticipate we will be doing some additional changes in the future so I want to add the attribution now